.env.example

```diff
@@ -1,3 +1,4 @@
-GROUP_ID="<bandada-group-id>"
-GROUP_API_KEY="<bandada-group-api-key>"
-MEMBER_INVITE_CODE="<bandada-member-invite-code>"
+# You can generate your API key from the Bandada Dashboard home page at
+# the following link https://bandada.pse.dev/groups.
+# Just activate the API key on the dashboard, copy and paste it here.
+ADMIN_API_KEY="<bandada-admin-api-key>"
```

Comment:
Can you add a space at the end of this file?

Reply:
sure, one sec.